### PR TITLE
MGMT-3088 push images to quay.io when we build from branch for instal…

### PR DESCRIPTION
**Pull request by oshercc**
…ler & agent

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -25,30 +25,40 @@ pipeline {
...
             sh '''docker push ${CONTROLLER_OCP}:${GIT_COMMIT}'''
         }
     }
+
+    stage('publish images on push to master') {
+        when {
+            branch 'master'
+        }
+        steps {
+            sh '''docker tag  ${INSTALLER} ${INSTALLER}:latest'''
```
Comment (gamli75) [inline Jenkinsfile:52]: you don't need 
withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
                sh '''docker login quay.io -u $USER -p $PASS'''
?
same as in line 33

Comment (oshercc) [reply Jenkinsfile:52]: ma?

**Review by gamli75 — APPROVED** [review]

**Review by YuviGold — APPROVED** [review]
LGTM

**Pull request merged**